sim/core/target.go

```diff
@@ -129,3 +132,19 @@ func (target *Target) GetMetricsProto(numIterations int32) *proto.TargetMetrics
 		Auras: target.auraTracker.GetMetricsProto(numIterations),
 	}
 }
+
+// ArmorDamageReduction currently assumes a level 70 attacker
+func (target *Target) ArmorDamageReduction() float64 {
```

Comment:
A better way to do this would be to have each target store its initial and current armor values, and have an 'AddArmor' function just like we do for stats. Then the current armor damage reduction value can be cached, and only updated when armor changes.